src/lint/lint.ts

```diff
@@ -45,10 +45,9 @@ export async function lint(options: Options, config: ProjectConfig) {
   const rules = lintLib.registry.getRules(ruleCodes || lintOptions.rules);
   const filter = new WarningFilter({
     warningCodesToIgnore: new Set(lintOptions.ignoreWarnings || []),
-    minimumSeverity: Severity.WARNING
+    minimumSeverity: options.minSeverity || lintOptions.minSeverity || Severity.WARNING,
```

Comment:
Hm, warning severity should really be a string enum rather than the numeric enum I used -.-

I guess there's still time for a breaking change there if we're very quick.

In any case, for now, you'll need to convert the string to the enum value, something like:

```typescript
function severityStringToValue(maybeSeverity: string): Severity|undefined {
  maybeSeverity = maybeSeverity.trim().toLowerCase();
  switch (maybeSeverity) {
    case 'error': return Severity.ERROR;
    case 'warning': return Severity.WARNING;
    case 'info': return Severity.INFO;
    default: return undefined;
  }
}
```

Then check for undefined and throw a useful error in before constructing the WarningFilter.